labs/lab-1-html-and-css/index.html

```diff
@@ -0,0 +1,71 @@
+<!doctype html>
```

Comment:
Your indentation is all OVER the place. 

In your editor, try Ctrl+shift+p and then type 'indent' to search for a command that'll automatically re-indent the whole file (you may need to select all, first. If you can't figure it out, bug me sometime tomorrow)